app/modules/annotations/models.py

```diff
@@ -263,14 +359,14 @@ def get_matching_set_default_query(self):
         # n.b. default will not take any locationId or ownership into consideration
         parts = {'filter': []}
 
-        # must have a content_guid (i.e. wbia guid)
-        parts['filter'].append(
-            {
-                'exists': {
-                    'field': 'content_guid',
-                }
-            }
-        )
+        # # must have a content_guid (i.e. sage guid)
```

Comment:
Is this filter still needed?  My guess is no because if an Annotation or Asset doesn't have a content_guid, we will create it as soon as we need to send it to Sage

Reply:
i am going to agree with this.  not coming up with a reason we would want Annotations to get sent to Sage and assigned a `content_guid`.   (i can think of some very edge-y cases for Assets; but that is irrelevant here.)

Reply:
Agreed, removing the commented-out filter